RInAppMessaging/Classes/Protocols/TaskSchedulable.swift

```diff
@@ -8,16 +8,16 @@ internal protocol TaskSchedulable: AnyObject {
     /// - Parameter milliseconds: Delay before running the task in milliseconds.
     /// - Parameter task: A block of code to run after the delay.
     /// - Parameter wallDeadline: Set `true` to use wall time (gettimeofday) instead of absolute time to ensure execution after returning from background
-    func scheduleWorkItem(milliseconds: Int, wallDeadline: Bool, closure: @escaping () -> Void)
+    func scheduleTask(milliseconds: Int, wallDeadline: Bool, _ task: @escaping () -> Void)
 }
 
 /// Default implementation.
 extension TaskSchedulable {
-    func scheduleWorkItem(milliseconds: Int, wallDeadline: Bool, closure: @escaping () -> Void) {
+    func scheduleTask(milliseconds: Int, wallDeadline: Bool, _ task: @escaping () -> Void) {
```

Comment:
Is this change so it can be called as a trailing closure?

Reply:
No, it's just about naming unification